cfpb/cfgov-refresh
# Add the option to include an extension in a phone number

We have a need to include phone extensions when adding a number to
contact details but the current field does not allow for the extra
characters. To avoid differences in formatting, we've added a new field
for the extension digits that are then automatically formatted on the
front end.

## Additions

- New `extension` option to the wagtail contact phone molecule
- New `ext` formatting on the front-end of the contact phone molecule

## Testing

1. `tox -e fast v1.tests.atomic_elements.organisms.test_organisms`
2. Build a sub-landing page and include an expandable.
3. Add contact phone molecule and include a phone number and extension.
4. Verify the extension is formatted on the page.

## Screenshots

![screen shot 2017-11-07 at 1 17 17 pm](https://user-images.githubusercontent.com/1280430/32512860-00b12ae0-c3be-11e7-8997-ddeb8bd4f66f.png)

![screen shot 2017-11-07 at 1 17 14 pm](https://user-images.githubusercontent.com/1280430/32512862-01fd38c6-c3be-11e7-9116-ea4d67cd2921.png)

## Checklist
